src/main.rs

```diff
@@ -102,6 +88,25 @@ Some useful options (to be used as `cargo fuzz run fuzz_target -- <options>`) in
  - `-only_ascii`: Only provide ASCII input
  - `-dict=<file>`: Use a keyword dictionary from specified file. See http://llvm.org/docs/LibFuzzer.html#dictionaries")
         )
+        .subcommand(fuzz_subcommand("cmin")
+             .about("Corpus minifier")
+             .arg(Arg::with_name("CORPUS")
+                  .help("directory with corpus to minify"))
+        )
+        .subcommand(fuzz_subcommand("tmin")
+             .about("Test case minifier")
+             .arg(Arg::with_name("runs").long("runs")
+                  .help("number of attempts to minimize we should make")
+                  .takes_value(true)
+                  .default_value("255")
```

Comment:
Is 255 enough to minimise anything non-trivial? 255 cases is seems like something that fuzzer could easily do in half a second so raising this value could be interesting.